fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflow.java

```diff
@@ -44,6 +47,12 @@
 @Singleton
 public class DruidWorkflow implements RequestWorkflowProvider {
 
+    private static final SystemConfig SYSTEM_CONFIG = SystemConfigProvider.getInstance();
+    private int druidUncoveredIntervalLimit = SYSTEM_CONFIG.getIntProperty(
```

Comment:
You can achieve what you want to do via changing the system config before constructing the DruidWorkflow.
```java
SystemConfig systemConfig = SystemConfigProvider.getInstance()
String uncoveredKey = SYSTEM_CONFIG.getPackageVariableName("druid_uncovered_interval_limit")

"myTest"()
setup:
systemConfig.setProperty(uncoveredKey, '5')
...
cleanup:
systemConfig.clearProperty()           
```